app/grandchallenge/core/utils/markdown.py

```diff
@@ -9,22 +12,64 @@ def extendMarkdown(self, md):  # noqa: N802
 
 
 class BS4Treeprocessor(Treeprocessor):
+
     def run(self, root):
+
+        el_class_dict = {
+            "img": "img-fluid",
+            "blockquote": "blockquote",
+            "table": "table table-hover table-borderless",
+            "thead": "thead-light",
+            "code": "codehilite",
+        }
+
         for el in root.iter():
-            if el.tag == "img":
-                el.set("class", "img-fluid")
 
-            elif el.tag == "blockquote":
-                el.set("class", "blockquote")
+            if el.tag in el_class_dict:
+                BS4Treeprocessor.set_css_class(el, el_class_dict[el.tag])
+
+        for i in range(len(self.md.htmlStash.rawHtmlBlocks)):
+
+            bs4block = BeautifulSoup(
+                self.md.htmlStash.rawHtmlBlocks[i], "html.parser"
+            )
+
+            el = bs4block.find()
+
+            if el and el.name in el_class_dict:
+                BS4Treeprocessor.set_css_class(el, el_class_dict[el.name])
+                self.md.htmlStash.rawHtmlBlocks[i] = str(bs4block)
+
+    @staticmethod
+    def set_css_class(element, class_name: str):
```

Comment:
```suggestion
    def set_css_class(*, element, class_name):
```

See https://peps.python.org/pep-3102/

No type annotations should be added as we're not checking them.

Reply:
Done

Reply:
Done